src/main/java/com/appland/appmap/output/v1/Parameters.java

```diff
@@ -53,12 +53,6 @@ public Parameters(CtBehavior behavior) {
     LocalVariableAttribute locals = (LocalVariableAttribute) codeAttribute.getAttribute(
         javassist.bytecode.LocalVariableAttribute.tag);
 
-    // We should be able to handle methods without debug
```

Comment:
I think this is quite a risky change.

I put this comment here because, when I previously made the change to continue after finding no locals, we saw many new test failures in the Land of Apps. These seemed to be happening because classes that had previously been skipped were now being instrumented. 

Unfortunately, I didn't record which projects showed these problems, and the few I've tried again were ok.

Reply:
If this is a too risky general change, can we limit it to only classes from the java.sql package? That would fix our inability to record SQL from the common DBs and also isolate the change to a subset that should be relatively easily testable.

Reply:
I'm having a hard time understanding how this change could introduce additional errors, aside from the fact that we're potentially hooking more code. 

Here's my take:
- The impact is already limited, as this change only introduces hooks to methods with no local variable table. In the wild, I've only ever seen this occur with SQL implementations, but it could also affect HTTP-related hooks. However, we don't rely on or reference these variables anywhere because it's expected that they will change between implementations.
- In my testing `null`, empty, and populated local variable tables are all hooked and recorded as expected. If this introduces an issue, I can't see this as the root of the problem. It would likely be some other side-effect introduced by code that was failing to be hooked prior to this.

I vote we move forward with releasing the change.

Reply:
Yeah, it was exactly the case that this caused code to be hooked that was previously being ignored. There were failures in the newly-hooked code.

I guess we'll see what happens.